webpack.common.js

```diff
@@ -10,9 +10,18 @@ module.exports = {
         ]
     },
     plugins: [
-        new CopyPlugin([{
-            from: "**/*",
-            to: "."
-        }])
+        new CopyPlugin([
+            {
+                from: "**/*",
+                to: "."
+            },
+            {
+                from: "../node_modules/libass-wasm/dist/subtitles-octopus-worker.*",
+                to: "JavascriptSubtitlesOctopus",
+                transformPath(targetPath, absolutePath) {
+                    return Promise.resolve(path.join("JavascriptSubtitlesOctopus", path.basename(targetPath)));
+                }
```

Comment:
I'm guessing this section was to accomodate the URL I mentioned above?

Reply:
I've removed the use of transformPath, and modified the options to reflect the new installation path of the worker script. I don't think we can completely avoid hard-coding /some/ path to the worker, unless it's installed at top level.

Reply:
That's fine, thanks for the improvements.